dev-aws/kafka-shared/customer-billing.tf

```diff
@@ -23,8 +23,39 @@ module "invoice_fulfillment_producer" {
 module "bills_total_api_consumer" {
   source = "../../modules/consumer"
 
-  topic = kafka_topic.invoice_fulfillment.name
+  topic          = kafka_topic.invoice_fulfillment.name
   consumer_group = "bills-total-api-reader"
 
-  cert_common_name = "customer-billing/bills-total-api" 
+  cert_common_name = "customer-billing/bills-total-api"
+}
+
+resource "kafka_topic" "invoice_fulfillment_deadletter" {
+  name               = "bex.internal.accountreadytobefulfilled_deadletter"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    # keep data for 7 days
+    "retention.ms" = "604800000"
```

Comment:
Can you increase the retention a little bit? This topic might need manual intervention and a week might be too rushed.